### ISSUE TITLE
ReactorExecutionListener is being fired twice when a serviceTask is being listened too

### ISSUE THREAD

**Issue by rmoskal** [camunda/camunda-bpm-reactor#84]
I've been your code for about a year now, and just hooked up serviceTasks. It works flawlessly with every other execution type I might be listening to.

This is with the latest release of camunda-bpm-reactor and camunda 7,8.

I can provide you with materials that will help you reproduce the error if it helps.

**Comment by jangalinski** [comment]
Yes, if there is a bug with the listeners, please provide an example to verify, thanks for reporting.

**Comment by rmoskal** [comment]
I've attached a project.  Here's the README,md:

You need java 1.8 and maven.  I'm sure you have it. 

Build it:

    mvn clean install
    
    
Runt it!

    java -jar target/camunda-1.0-SNAPSHOT.jar

You should see something like this in the logs:

    2018-06-21 12:57:21.139 ERROR 2357 --- [           main] com.stripmall.bpm.ServiceTaskListener    : I am here---------------------29b713b9-7574-11e8-b5bb-922014a9f425
    2018-06-21 12:57:21.140 ERROR 2357 --- [           main] com.stripmall.bpm.ServiceTaskListener    : I am here---------------------29b713b9-7574-11e8-b5bb-922014a9f425


You can toggle back and forth in the main class:  com.stripmall.bpm.RestApplication postDeployment listener to try a userTask. The listener is only called once then!


	@EventListener
	public void processPostDeploy(PostDeployEvent event) {
		runtimeService.startProcessInstanceByKey("sandbox_servicetask1");
        //runtimeService.startProcessInstanceByKey("sandbox_userTask");
	}


[reactor-sample.zip](https://github.com/camunda/camunda-bpm-reactor/files/2124682/reactor-sample.zip)

**Comment by rmoskal** [comment]
Jan: Is there a high or low likelihood of you getting to look at this?

**Comment by jangalinski** [comment]
Very busy at the moment.  I will look into it, but can’t promise for this week

**Comment by rmoskal** [comment]
OK. Thanks.

**Comment by jangalinski** [comment]
Thanks for reporting, I had a look into it. As soon as we have a error event on the task (no matter if service- or user), the executionLIstener is called twice. I guess this is because in the camunda execution graph there are two entries (one for the task, one for the error event) ...  you can reproduce this when you add an error event to the user task and change the task type on the ServiceTaskLIstener to "userTask" ....

I will flag this as bug.
